### ISSUE TITLE
Make CodeMirror more accessible

### ISSUE THREAD

**Issue by MarcoZehe** [TryGhost/Ghost#11871]
### Issue Summary

The CodeMirror instance being used for cards like HTML or Code is not accessible to screen readers. It uses a hidden textarea for input, but clears it immediately as soon as the items are being transferred to the contentEditable portion that does the actual displaying.

Firefox [solved this](https://bugzilla.mozilla.org/show_bug.cgi?id=919711) by including Chrome's AccessibleTextarea plugin. That plugin always displays the currently displayed/edited line in the textarea, and actually forwards the polling to another hidden textarea it keeps blank, as CodeMirror likes it.

The [`inputStyle`](https://codemirror.net/doc/manual.html#option_inputStyle) option is then set to that plugin and it is included in the webpack config so it gets built into the minified CodeMirror asset that is then actually being loaded.

[here](https://phabricator.services.mozilla.com/D24930) is the Phabricator revision with the full diff view, including the bundled asset.

Both Firefox and Chrome ship this in their developer tools, Firefox since version 68, and Chrome even longer. This is for CodeMirror 5, since the natively accessible [version 6](https://codemirror.net/6/) is still in development.

I propose to include this plugin in Ghost's admin client for the various times CodeMirror is being used: in the Koenig editor, as well as the integrations and code injection screens.

### To Reproduce

1. With NVDA on Windows and Firefox, open a post and insert an HTML card.
2. Try to navigate the card using the arrow keys *and* get speech output. It will fail.

Without the above proposed fix, screen reader users cannot edit cards, code injections etc., that use CodeMirror.

### Technical details:

* Ghost Version: 3.17.1
* Node Version:
* Browser/OS: Firefox on Windows 10, Chrome on Windows 10 or Mac.
* Database: Whatever.

CC @JohnONolan @daviddarnes

**Comment by MarcoZehe** [comment]
This issue is still present in current Ghost.

**Comment by rkingett** [comment]
This issue is still present, even in 4.0.

**Comment by rkingett** [comment]
This should still be orked on.
